server/services/store/sqlstore/blocks.go

```diff
@@ -799,7 +745,7 @@ func (s *SQLStore) replaceBlockID(db sq.BaseRunner, currentID, newID, workspaceI
 }
 
 func (s *SQLStore) duplicateBlock(db sq.BaseRunner, boardID string, blockID string, userID string, asTemplate bool) ([]model.Block, error) {
-	blocks, err := s.getSubTree3(db, boardID, blockID, model.QuerySubtreeOptions{})
+	blocks, err := s.getSubTree2(db, boardID, blockID, model.QuerySubtreeOptions{})
```

Comment:
The only possible concerning part is this one. But I think it is correct. It basically gets the subtree of 2 now that the only block containing other blocks is the cards, so... looks safe to search for only the block and his children.